pkg/features/feature.go

```diff
@@ -11,6 +11,7 @@ type feature struct {
 	envVar       string
 	name         string
 	defaultValue bool
+	unchangeable bool
```

Comment:
I read the PR description and the intent is to leave it to customers to set environment variables (in case of ACS CS, we'll have to figure that somehow per tenant) at this point. The idea of `unchangeable` feature is the one where the actual environment variable does not matter, a default is always taken. I do still think that `unchangeable` is not a good name. Let me think more on the suggestion.

Reply:
My suggestion would be to call this attribute `environmentValueRespected` (or similarly). I also suggest that function `registerFeature()` sets `environmentValueRespected` to `!buildinfo.ReleaseBuild` and `registerChangeableFeature()` (or how you want to call it) sets `environmentValueRespected` to `true`.